lighthouse-core/gather/gather-runner.js

```diff
@@ -68,18 +68,15 @@ class GatherRunner {
    * never fired on it.
    * @param {Driver} driver
    * @param {string=} url
-   * @param {number=} duration
    * @return {Promise<void>}
    */
   static async loadBlank(
       driver,
-      url = constants.defaultPassConfig.blankPage,
-      duration = constants.defaultPassConfig.blankDuration
+      url = constants.defaultPassConfig.blankPage
```

Comment:
can this go back to being one line now :)

Reply:
yep! 🎉